### PR TITLE
Create prefix.tag

### PR DESCRIPTION
**Pull request by Greaper88**

### CODE REVIEW

File: src/main/resources/tags/info/prefix.tag [hunk 1]
```diff
@@ -0,0 +1,9 @@
+type: text
+
+---
+
+The prefix is added by Floodgate to indicate that the player is Bedrock; and more importantly, to minimize the possibility of player name conflict issues.
+
+We strongly recommend that you **do not remove** the prefix, but instead change it.
+
+You can change the prefix (`.`) in players' names by editing the option in Floodgate's config.yml file. Characters that a Java player cannot normally choose to be part of hteir names are recommended.
```
Comment (Chew) [inline src/main/resources/tags/info/prefix.tag:9]:
```suggestion
You can change the prefix (`.`) in players' names by editing the option in Floodgate's config.yml file. Characters that a Java player cannot normally choose to be part of their names are recommended.
```

Comment (Greaper88) [reply src/main/resources/tags/info/prefix.tag]: whoops. lol

File: src/main/resources/tags/info/prefix.tag [hunk 2]
```diff
@@ -0,0 +1,9 @@
+type: text
+
+---
+
+The prefix is added by Floodgate to indicate that the player is Bedrock; and more importantly, to minimize the possibility of player name conflict issues.
+
+We strongly recommend that you **do not remove** the prefix, but instead change it.
+
+You can change the prefix (`.`) in players' names by editing the option in Floodgate's config.yml file. Characters that a Java player cannot normally choose to be part of their names are recommended.
```
Comment (Camotoy) [inline src/main/resources/tags/info/prefix.tag:9]: Add such characters: `+`, `-`, `!`, `*` I think are the general ones.

**Comment by Camotoy** [comment]
I added a warning on what could happen if a Bedrock and Java player join the server at the same time with the same name.

**Review by Camotoy — APPROVED** [review]

**Pull request merged**